README.md

```diff
@@ -69,6 +71,8 @@ After writing this method, our new result should be:
 
 **think**: Where should we call `strip_punctuation`? Does it matter? Why?
 
+I call it inside the analyze_mood method at the top so that the rest of the method uses the correct version of words.
+
```

Comment:
But why is it better to call it here instead of just before the method entirely? You called it a fine place. I want to know why inside the method is better than outside the method.